lib/judge/config.rb

```diff
@@ -8,24 +8,24 @@ class Config
     @@ignore_unsupported_validators = false
 
     def expose(klass, *attributes)
-      attrs = (@@exposed[klass] ||= [])
+      attrs = (@@exposed[klass.name] ||= [])
       attrs.concat(attributes).uniq!
     end
 
     def exposed
-      @@exposed
+      @@exposed.map { |k, v| [k.constantize, v] }.to_h
```

Comment:
Okay, sounds good to me that we fix the tests instead, given that `exposed` is an internal API so it doesn't matter that it's not returning the same type (i.e. string) as what it was getting (i.e. class).

@joecorcoran the PR aims to maintain the exact same usage, so it's meant to be used *only* with a model class, e.g. `expose Post, :title, :body`. Developers don't need to know that keys are stored as strings now.

@jamesmk I assume that you'll be taking this PR forward? Let me know if there's anything I can help.

Thanks